pkg/mounter/proxy/server/ossfs2/driver.go

```diff
@@ -142,6 +157,10 @@ func (h *Driver) Mount(ctx context.Context, req *proxy.MountRequest) error {
 	})
 
 	if err == nil {
+		// Handle mount result
+		server.SafeHandleMountResult(monitor, cmd, err)
```

Comment:
Quite confuse about handling mount results when err is nil versus when it's not.